src/main/java/edu/ucsb/cs156/happiercows/repositories/AnnouncementRepository.java

```diff
@@ -0,0 +1,9 @@
+package edu.ucsb.cs156.happiercows.repositories;
+
+import org.springframework.data.repository.CrudRepository;
+
+import edu.ucsb.cs156.happiercows.entities.Announcement;
+
+public interface AnnouncementRepository extends CrudRepository<Announcement, Long> {
```

Comment:
This needs to be annotated with the `@Repository` annotation (see the other repository classes in the same directory.